eldritch/test_events.py

```diff
@@ -149,8 +156,11 @@ def advance_turn(self, target_turn, target_phase):
           break
         continue
       if self.state.turn_phase == "upkeep":
-        self.assertIsInstance(self.state.event_stack[-1], events.SliderInput)
-        self.state.event_stack[-1].resolve(self.state, "done", None)
+        # self.assertIsInstance(self.state.event_stack[-1], events.SliderInput)
```

Comment:
Having items that are usable during upkeep breaks the `assertIsInstance(event, SliderInput)`. Maybe I should have an optional `may_break_early=False` arg that, if False will skip past SliderInputs, but not other Usables/chocies.

Reply:
Oh, I see. The story becomes usable because you roll the die. You could just put something like `if not isinstance(SliderInput): self.state.done_using[char_idx] = True; continue` in here.